src/lib/data.ts

```diff
@@ -35,7 +30,9 @@ fs.readFile('./data.db', (err, file) => {
 
   domainToMapping = createDomainCache(data.mappings)
   idToMapping = createIdCache(data.mappings)
-})
+} catch (err) {
+  console.error(`Error reading from data.db on startup, ${err}`)
```

Comment:
I think `console.log` is better, because we expect this to happen when it starts.... 
```
console.log(	
      'File does not exist, but do not worry. File will be created on first save',	
      err	
    )
```